Description: Fix warning on 64 bit machines. Explicitely set variables as
 unsigned ints.
Origin: http://git.debian.org/?p=pkg-multimedia/lame.git;a=blob;f=debian/patches/07-field-width-fix.patch
Forwarded: commit:1.282
Applied-Upstream: commit:1.282

--- a/frontend/parse.c
+++ b/frontend/parse.c
@@ -372,11 +372,11 @@
     const char *b = get_lame_os_bitness();
     const char *v = get_lame_version();
     const char *u = get_lame_url();
-    const size_t lenb = strlen(b);
-    const size_t lenv = strlen(v);
-    const size_t lenu = strlen(u);
-    const size_t lw = 80;       /* line width of terminal in characters */
-    const size_t sw = 16;       /* static width of text */
+    const unsigned int lenb = strlen(b);
+    const unsigned int lenv = strlen(v);
+    const unsigned int lenu = strlen(u);
+    const unsigned int lw = 80;       /* line width of terminal in characters */
+    const unsigned int sw = 16;       /* static width of text */
 
     if (lw >= lenb + lenv + lenu + sw || lw < lenu + 2)
         /* text fits in 80 chars per line, or line even too small for url */
